### DISCUSSION TITLE
lotus v1.8.0 and network v12 upgrade discussion

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/lotus#6082] [General Q&A] [answered]
If you have any questions wrt the[ lotus v1.8.0 release](https://github.com/filecoin-project/lotus/discussions/6084) and network [v12 upgrade](https://github.com/filecoin-project/community/discussions/74#discussioncomment-642326), leave a comment here!

> Note: Per [the new release philosophy](https://github.com/filecoin-project/lotus/discussions/5889#discussioncomment-602796) the lotus team is practicing, which uses an **even** minor version to indicate a network upgrade. Therefore, this upcoming version will be **v1.8.0** that being developed based off [lotus v1.6.0](https://github.com/filecoin-project/lotus/releases/tag/v1.6.0). lotus **v1.8.0** will integrate actor v4, supports network sync improvements for **network v12**.  The current `v1.7.0-rc1` will be bumped to `v1.9.0-rc1`, which includes everything from `v1.7.0-rc1` and new codes for v1.8.0 to support the network v12 upgrade. The master branch version will be updated to `v1.11.0-dev`.

**Comment by William8Work** [comment]
just to confirm, @jennijuju, do you mean the v1.8.0 will be available on April 26, and we need to get upgraded before 2021-04-28T22:00:00Z, network v12 upgrade epoch 711360?

**Reply by jennijuju** [reply]
yes!

**Reply by Jasonwu123** [reply]
Is it a forced upgrade？

**Comment by Fatman13** [comment]
What's making it a consensus breaking upgrade? Any change list?

**Reply by jennijuju** [reply]
More details will be available when the releases are tagged next Monday.

**Comment by joshua-ne** [comment]
It’s kind of VERY rush, we DO need more time for code integration and testing. @jennijuju

**Reply by jennijuju** [reply]
Although I totally understand this is not ideal and is a very rare occasion, the lotus team is working hard on testing and validation to have the release out as soon as possible. All impls are also minimizing the code changes to make patches quick to integrate. However, it’s important that all miners have setups that are able to upgrade quickly and responsively

**Comment by firesWu** [comment]
we need more time to testing and upgrading our cluster

**Reply by jennijuju** [reply]
see https://github.com/filecoin-project/lotus/discussions/6082#discussioncomment-646485

**Comment by psh0691** [comment]
Why did you skip v1.7.0?

**Reply by jennijuju** [reply]
It's to practice the new lotus release/version philosophy that is explained [here](https://github.com/filecoin-project/lotus/discussions/6084#discussion-3331970)

**Comment by jennijuju** [comment]
We heard your concern and made some small changes, hopefully, it helps! https://github.com/filecoin-project/lotus/discussions/6084#discussioncomment-651639

**Comment by hhj517** [comment]
Fixed V1 bug?
lotus-miner sectors extend --v1-sectors
ERROR: mpool push message: GasEstimateMessageGas error: estimating gas used: message execution failed: exit 16, reason: invalid expiration 1661347, total sector lifetime (1555594) cannot exceed 1555200 after activation 105753 (RetCode=16)

**Comment by SgtCoin** [comment, marked as answer]
Archiving for future reference due to age. Network is now on v1.13+
